Library/Formula/justrelease.rb

```diff
@@ -0,0 +1,19 @@
+class Justrelease < Formula
+  homepage "github.com/justrelease"
```

Comment:
Please add `https://` in front of this URL. Also please add a short description above this line using `desc "..."`.